.github/workflows/platform-executables-acceptance-tests.yml

```diff
@@ -40,35 +40,41 @@ jobs:
           git fetch --prune --unshallow
           echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
 
-  wait-for-oclif-releases:
-    name: Wait for oclif platform executables Release to complete
-    runs-on: ubuntu-latest
+  wait-for-brew-releases:
+    name: Wait for brew platform executables Release to complete
```

Comment:
This is not correct. It's a brew test, not brew platform executables. Similarly, now that we are adding all acceptance tests in to this one file, we should rename it to just acceptance-tests.yml

Reply:
Make sense, Will rename this file